test/JWT.hs

```diff
@@ -74,7 +76,7 @@ spec = do
 
     describe "with an Expiration Time claim" $ do
       describe "when the current time is prior to the Expiration Time" $ do
-        let now = utcTime "2010-01-01 00:00:00"
+        let now = utcTime "2011-03-01 00:00:00"
```

Comment:
it needs otherwise `iat` validation will failed or we could turn off `iat` validation for those cases.
slightly prefer to update the value.